### ISSUE TITLE
sab 1.2.x -- cherrypy 7.x upgrade

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#640]
i swear they have some people from firefox on their team now.. 

https://github.com/cherrypy/cherrypy/releases
```
7.1.0
-----

# 1458: Implement systemd's socket activation mechanism for
  CherryPy servers, based on work sponsored by Endless Computers.

  Socket Activation allows one to setup a system so that
  systemd will sit on a port and start services
  'on demand' (a little bit like inetd and xinetd
  used to do).

7.0.0
-----

Removed the long-deprecated backward compatibility for
legacy config keys in the engine. Use the config for the
namespaced-plugins instead:

 - autoreload_on -> autoreload.on
 - autoreload_frequency -> autoreload.frequency
 - autoreload_match -> autoreload.match
 - reload_files -> autoreload.files
 - deadlock_poll_frequency -> timeout_monitor.frequency

6.2.1
-----

# 1460: Fix KeyError in Bus.publish when signal handlers
  set in config.

6.2.0
-----

* #1441: Added tool to automatically convert request
  params based on type annotations (primarily in
  Python 3). For example:

    @cherrypy.tools.params()
    def resource(self, limit: int):
        assert isinstance(limit, int)

6.1.1
-----

* Issue #1411: Fix issue where autoreload fails when
  the host interpreter for CherryPy was launched using
  ``python -m``.

6.1.0
-----

* Combined wsgiserver2 and wsgiserver3 modules into a
  single module, ``cherrypy.wsgiserver``.

6.0.2
-----

* Issue #1445: Correct additional typos.
```

**Comment by Safihre** [comment]
Seems there's absolutely nothing of value for us? :smile:

**Comment by thezoggy** [comment]
@Safihre not that I can tell, why its for 1.2.x..

**Comment by Safihre** [comment]
They throw version numbers around like it's free candy, already preparing for 8.0.0:
https://github.com/cherrypy/cherrypy/blob/master/CHANGES.txt

**Comment by shypike** [comment]
Yeah.
I think the current versions with patches works fine.
I'm never in a hurry to update CP. Too many subtle side-effects.

**Comment by thezoggy** [comment]
i agree, just tossing it up here so its on the radar to check out in case they do add something we want/need.. or do more native bug fixes..

**Comment by Safihre** [comment]
Closed by #677
